mitmproxy/net/websockets/utils.py

```diff
@@ -88,3 +89,26 @@ def get_client_key(headers):
 
 def get_server_accept(headers):
     return headers.get("sec-websocket-accept", None)
+
+
+def make_extension(extension):
```

Comment:
I don't think it will work because we need to pass the extension(initialized with correct params) as arg to the WSConn object